### PR TITLE
Added randomizer for non-comic days (comics come out on mon-wed-fri). Size restraint.

### PR DESCRIPTION
**Pull request by RedNax67**
As per the subject line.

### CODE REVIEW

**Comment by RedNax67** [comment]
More fixes

File: DailyXKCD.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ Module.register("DailyXKCD", {
         var xkcd = document.createElement("img");
         xkcd.src = this.dailyComic;
         if (this.config.invertColors) {
-            xkcd.setAttribute("style", "-webkit-filter: invert(100%);")
+                 xkcd.setAttribute("style", "-webkit-filter: invert(100%); max-width: 100%; max-height: 100%; height: 280px; ")
```
Comment (Blastitt) [inline DailyXKCD.js:60]: Looks like the style changes only take effect when the invertColors config option is true.

File: node_helper.js [hunk 2]
```diff
@@ -18,13 +18,27 @@ module.exports = NodeHelper.create({
...
+                        var comic = JSON.parse(body);                         
+                        var rndcomic = Math.floor((Math.random() * comic.num) + 1); 
+                        var rndUrl = "http://xkcd.com/" + rndcomic.toString() + "/info.0.json";
+                        console.log("Random url: " + rndUrl);
+                        request(rndUrl, function (error, response, body) {
+                            if (!error && response.statusCode == 200) {
+                                self.sendSocketNotification("COMIC", JSON.parse(body));
+                            }
+                        });
+					}
```
Comment (Blastitt) [inline node_helper.js:38]: Code looks good, but please fix indentation.

**Comment by Blastitt** [comment]
How extensively have you tested this? The new features are cool, but does the size limiting work? Does that parseFloat stuff work?

**Comment by Blastitt** [comment]
Thank you for your interest. You have failed to update this pull request and your proposed changes have been addressed in another pull request, so I am closing this one.

**Pull request closed without merging**